wagtail_localize_rws_languagecloud/tests/helpers.py

```diff
@@ -44,3 +50,32 @@ def create_test_project_settings(translation_source, translations, **settings_da
     return LanguageCloudProjectSettings.get_or_create_from_source_and_translation_data(
         translation_source, translations, **data
     )
+
+
+def create_editor_user(username="testeditor"):
```

Comment:
Worth noting here that we also have a slightly different function to help with creating test users

https://github.com/zerolab/wagtail-localize-rws-languagecloud/blob/ad7f8c94ead3eef43af6a21544e26353f39b473d/wagtail_localize_rws_languagecloud/tests/test_emails.py#L14-L20

We don't need to address it right now, but as you work on this sprint there might be an opportunity to pull out some nice generic helpers here. Maybe one to have in mind if another ticket comes up that requires creating test users with various permissions.

Reply:
Usually we'd use factory-boy for generating test data, including users. It currently isn't being used on the project. Would you be OK for me to set that up? (On future PRs)

Reply:
Yeah if you've got time to do that in the scope of this work, having some nice factories would be great :+1: